Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -1642,11 +1642,21 @@ open class NavigationMapView: UIView {
         if let image = Bundle.mapboxNavigation.image(named: "RouteInfoAnnotationRightHanded") {
             // define the "stretchable" areas in the image that will be fitted to the text label
             // These numbers are the pixel offsets into the PDF image asset
-            let stretchX = [ImageStretches(first: Float(33), second: Float(52))]
-            let stretchY = [ImageStretches(first: Float(32), second: Float(35))]
+            var stretchX: [ImageStretches]!
+            var stretchY: [ImageStretches]!
             // define the "content" area of the image which is the portion that the maps sdk will use
             // to place the text label within
-            let imageContent = ImageContent(left: 34, top: 32, right: 56, bottom: 50)
+            var imageContent: ImageContent!
+            
+            if !UIDevice.current.hasNotch {
```

Comment:
Would it make sense to replace `UIDevice.current.hasNotch` with actual top and bottom safe area inset values? They'll be 0 for devices without notch anyway.